src/bin/analyzer.js

```diff
@@ -113,7 +114,7 @@ if (mode === 'server') {
     excludeAssets,
     logger: new Logger(logLevel)
   });
-} else {
+} else if (mode === 'static') {
```

Comment:
We're only extending the usage, thus IMO that we should kept `static` as default as usual.

Reply:
Okay~